src/app/components/navbar/index.html

```diff
@@ -5,7 +5,12 @@
 >
 
   <sky-demo-page-summary>
-    The navbar component displays a list of top-level navigation items. Navigation items can include behavior such as displaying sub-navigation items in dropdown menus with the <a routerLink="../dropdown">dropdown component</a>.
+    <p>
+      The navbar component displays a list of top-level navigation items. Navigation items can include behavior such as displaying sub-navigation items in dropdown menus with the <a routerLink="../dropdown">dropdown component</a>.
+    </p>
+    <p>
+      Blackbaud developers should use the omnibar for top-level navigation instead of the navbar component. <a stacheRouterLink="https://docs.blackbaud.com/omnibar-docs/">The omnibar</a> is for internal Blackbaud use only.
```

Comment:
This needs to use `href` instead of `stacheRouterLink`. 

What are your thoughts on linking the action of the first sentence instead?

```
Blackbaud developers should <a href="https://docs.blackbaud.com/omnibar-docs">use the omnibar</a> for top-level navigation instead of the navbar component.  The omnibar is for internal Blackbaud use only.
```

Reply:
Good catch on the href. I copied and pasted and didn't fix that. Sorry.

As for putting the link in the first sentence, that's what I initially had. But then I second-guessed it and thought that maybe it should be in the second sentence that explains that it's for internal use only so that third-party developers won't click it and be frustrated. ... But the fact that you noticed it and thought it should be in the first sentence makes me think it was the wrong call. I'll change it back and fix the a tag to use `href`.